tests/rip7560/rip7560TestUtils.go

```diff
@@ -0,0 +1,122 @@
+package rip7560
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/core/vm"
+	"github.com/ethereum/go-ethereum/internal/ethapi"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/status-im/keycard-go/hexutils"
+	"math/big"
+	"os"
+	"testing"
+)
+
+const PREDEPLOYED_SENDER = "0xed0a7aabc745fe5d8bc6ad625c767df86044d049"
```

Comment:
This address is usually overridden anyways, let's make it `0x777777....777777` for clarity?